crates/context/config/src/client/env/proxy/types/evm.rs

```diff
@@ -308,7 +311,8 @@ impl From<ProposalAction> for SolProposalAction {
                     amount,
                 } => {
                     let data = TransferData {
-                        recipient: receiver_id.parse().expect("Invalid address"),
+                        recipient: Address::from_str(&receiver_id)
```

Comment:
like this.... potentially dangerous, because this will cause the money to be burnt, whenever the address is invalid, which is questionable

Reply:
Added error exit if it fails to convert to address